automation/src/test/scala/org/broadinstitute/dsde/test/api/WorkspaceApiSpec.scala

```diff
@@ -100,6 +100,40 @@ class WorkspaceApiSpec extends TestKit(ActorSystem("MySpec")) with AnyFreeSpecLi
         Rawls.billingV2.deleteBillingProject(billingProjectName)
       }
 
+      "to create a v2 workspaces (in a v2 billing project) with properly updated labels and display name" in {
```

Comment:
Do we need this test?  This test suite already tests that we can create projects and `WorkspaceServiceSpec` already tests that the labels and display name are set correctly.  I don't see this test adding value or covering functionality that we don't already have covered.

If you _do_ think we need this test, then:

nit: for the sake of making progress, I don't think we should do this now, but I think this test should be named differently and located elsewhere.  I would rather call this test something like, "Rawls should set labels on the underlying Google Project when creating a new Workspace".  I feel like these clearly articulates the logic under test.  As it is currently written, it seems to me that this is a test about Rawls Billing Project owners and what they can do.  The fact that they can create Workspaces, to me, is one test.  The fact that when we create Workspaces, labels (and display name) are set is a totally different test.  

Again, it's a nit, no need to change it now, just something to think about as we create and organize tests.

Reply:
I was able to set up the test name to what you suggested and the test seems happy. It seemed easy enough to move now so I opted to do that instead of waiting, in case we forgot.

(We talked about this in office time: I think we do need this test. The main difference between this test and the other tests is that this is an integration test. I think it provides value in that we don't really validate labels on a real project anywhere else. This test will ensure that we update real projects correctly. It saved me the hassle of finding the project in GCP console to verify if the project was updated correctly and should ensure that our logic for updating the project doesn't break in the future.)